card/mod/search/template/abstract/03_filter/filter_form.haml

```diff
@@ -15,27 +15,29 @@
         %input{ type: :hidden, name: "filter[not_ids]", class: "_not-ids", value: not_ids }
         .d-flex.flex-row
           - if sort_input_field
-            %i.material-icons.text-muted.md-24.p-2 sort
-            .input-group.sort-input-group.mb-2.card-outline-primary.mr-2
-              %span.input-group-addon sort
-              = sort_input_field
+            .sort-in-filter-form
+              %i.material-icons.text-muted.md-24.p-2.float-left sort
+              .input-group.sort-input-group.mb-2.card-outline-primary.mr-2
```

Comment:
replace `.mr-2` with `.mr-5` so that the zoom lens icon will be visible in browse pages